src/Test.UnitTests.Sarif/TestData/InsertOptionalDataVisitor/ExpectedOutputs/CoreTests-Relative_All.sarif

```diff
@@ -68,14 +68,16 @@
         {
           "repositoryUri": "https://REMOTE_URI",
           "mappedTo": {
-            "uri": "file:///REPLACED_AT_TEST_RUNTIME",
```

Comment:
>, [](start = 53, length = 1)

Because of the change in ordering of operations in the visitor -- namely, the creation of `VersionControlProvenance` happens at the end, _after_ all the `ArtifactLocation`s have been visited -- we no longer visit this `ArtifactLocation` and therefore we do not add an `Artifact` for it. IMO that's just fine.